presto-common/src/test/java/com/facebook/presto/common/type/TimeZoneKeyTest.java

```diff
@@ -1,140 +1,123 @@
-    String zoneId = "etc/universal+08:00";
-    String normalizedZoneId = (String) normalizeZoneId.invoke(timeZoneKey, zoneId);
-    assertEquals(normalizedZoneId, "+08:00");
-
-    zoneId = "etc/universal-08:00";
-    normalizedZoneId = (String) normalizeZoneId.invoke(timeZoneKey, zoneId);
-    assertEquals(normalizedZoneId, "-08:00");
-
-    zoneId = "etc/universal+11:23";
-    normalizedZoneId = (String) normalizeZoneId.invoke(timeZoneKey, zoneId);
-    assertEquals(normalizedZoneId, "+11:23");
-  }
-
-  @Test
-  @Named("UCT TEST")
-  public void uctZoneIdTest() throws InvocationTargetException, IllegalAccessException {
-    String zoneId = "etc/uct+06:00";
-    String normalizedZoneId = (String) normalizeZoneId.invoke(timeZoneKey, zoneId);
-    assertEquals(normalizedZoneId, "+06:00");
-
-    zoneId = "etc/uct-10:10";
-    normalizedZoneId = (String) normalizeZoneId.invoke(timeZoneKey, zoneId);
-    assertEquals(normalizedZoneId, "-10:10");
-
-    zoneId = "etc/uct+1";
-    normalizedZoneId = (String) normalizeZoneId.invoke(timeZoneKey, zoneId);
-    assertEquals(normalizedZoneId, "+01:00");
-  }
-
-  @Test
-  @Named("ZULU TEST")
-  public void zuluZoneIdTest() throws InvocationTargetException, IllegalAccessException {
-    String zoneId = "etc/zulu+06:00";
-    String normalizedZoneId = (String) normalizeZoneId.invoke(timeZoneKey, zoneId);
-    assertEquals(normalizedZoneId, "+06:00");
-
-    zoneId = "etc/zulu-10:10";
-    normalizedZoneId = (String) normalizeZoneId.invoke(timeZoneKey, zoneId);
-    assertEquals(normalizedZoneId, "-10:10");
-
-    zoneId = "etc/zulu+1";
-    normalizedZoneId = (String) normalizeZoneId.invoke(timeZoneKey, zoneId);
-    assertEquals(normalizedZoneId, "+01:00");
-  }
-
-  @Test
-  @Named("EXCEPTION TEST")
-  public void exceptionZoneIdTest() {
-    String[] invalidTimeZones = {"GMT-13:00", "etc/6", "etc/*", "etc/", "etc/gmt+24:00", "etc/gmt+01",
-            "etc/gmt+01:000", "etc/utc+27", "etc/utc-13", "etc/gmt-16", "etc/greenwich+01",
-            "etc/universal+01", "fsdkjflksdflsdlfj", "ETC/+06:00", "ETC/06:00", "ETC/+6"};
-
-    for (String timeZone : invalidTimeZones) {
-      try {
-        normalizeZoneId.invoke(timeZoneKey, timeZone);
-        fail("Expected TimeZoneNotSupportedException to be thrown for " + timeZone);
-      } catch (InvocationTargetException e) {
-        Throwable cause = e.getCause();
-        assertTrue(cause instanceof TimeZoneNotSupportedException,
-                "Expected TimeZoneNotSupportedException, but got " + cause.getClass().getSimpleName());
-      } catch (Exception e) {
-        fail("Unexpected exception type: " + e.getClass().getSimpleName());
-      }
-    }
-  }
-}
+ import org.testng.annotations.Test;
+
+ import javax.inject.Named;
+
+ import static org.testng.Assert.*;
+
+ public class TimeZoneKeyTest {
+
+   @Test
+   @Named("GMT TEST")
+   public void gmtZoneIdTest() {
```

Comment:
Please look at CheckStyle output (maven-checks), and see README.md for formatting suggestings.  It looks like your formatting is off.